### DISCUSSION TITLE
Suggest enabling GitHub Discussions for the zigpy organization and disable/close GitHub Discussions in each other indiviudal code repository to consolidate all discussion to one place

### DISCUSSION THREAD

**Discussion by Hedda** [zigpy/zigpy#1437] [General]
I like to suggest you enable GitHub Discussions for the zigpy organization in order to consolidate all zigpy/zha discussions to a single place:

* https://github.com/zigpy

There enable https://github.com/orgs/zigpy/discussions but make it use (redirect) to existing https://github.com/zigpy/zigpy/discussions

I understand that you now able to choose to use this existing repository or create a repository specifically to hold your organization discussions. Meaning that have to option to simply point/redirect to this existing zigpy disscussions section.

* https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization

    "_**You can use GitHub Discussions in an organization as a place for your organization to have conversations that aren't specific to a single repository within your organization.**_

_When you enable organization discussions, **you will choose a repository in the organization to be the source repository for your organization discussions. You can use an existing repository or create a repository specifically to hold your organization discussions.** Discussions will appear both on the discussions page for the organization and on the discussion page for the source repository._"

Follow-up is disabling or closing all the other GitHub Discussions in each of the other individual code repositories that have discussions:

* https://github.com/zigpy/zigpy/discussions
* https://github.com/zigpy/zha/discussions
* https://github.com/zigpy/zha-device-handlers/discussions
* https://github.com/zigpy/zigpy-cli/discussions
* https://github.com/zigpy/quirk-generator/discussions
* https://github.com/zigpy/zigpy-znp/discussions
* https://github.com/zigpy/zha-websocket-server/discussions
* https://github.com/zigpy/zigpy-xbee/discussions

That way the single discussions section will appear at the top level and there will only be one discussions section which consolidates all discussion posts to a single location, and it will make it easier for everyone to not miss discussions when they are no longer spread out.

<img width="1310" height="771" alt="image" src="https://github.com/user-attachments/assets/8414fd1a-8b82-49b6-a3f8-a87f2a2b6000" />

**Comment by Hedda** [comment]
Bump and ping @missyquarry in [her role as Community Manager for Home Assistant (and by extension also the Open Home Foundation)](https://community.home-assistant.io/t/an-introduction-your-new-community-social-media-manager/693623).

Tip, if want to see a good proof-of-concept example of this implemented then check out Music Assistant's discussions section which is only exposing one consolidated discussion section organization level at https://github.com/music-assistant that gives them https://github.com/orgs/music-assistant/discussions which they link/redirect from a single repo https://github.com/music-assistant/support (so the discussions section used is actually linked from https://github.com/orgs/music-assistant/discussions ):

<img width="964" height="1043" alt="image" src="https://github.com/user-attachments/assets/2f901e9c-a9c5-4577-9641-762900bcd9e7" />

**Reply by TheJulianJES** [reply]
Unlike Music Assistant, our repositories are for more different projects, so I don't think having them collected in one place makes much sense for us.

For example, there are other projects/people that only use zigpy, but not through ZHA or HA. But most users on GitHub won't care if someone has a question on how to use some internal zigpy API – it's something we likely need to answer anyway.

Looking at the amount of discussions we have in the individual repositories now already, discoverability doesn't seem like an issue.

Or are there more benefits you'd see in having them combined in one place AND disabling discussions in all individual repos?
